src/network-manager/azext_network_manager/aaz/latest/network/manager/ipam_pool/_create.py

```diff
@@ -72,10 +72,9 @@ def _build_arguments_schema(cls, *args, **kwargs):
             arg_group="Body",
             help="The geo-location where the resource lives",
             required=True,
-            # todo: confirm
-            # fmt=AAZResourceLocationArgFormat(
-            #     resource_group_arg="resource_group",
-            # ),
+            fmt=AAZResourceLocationArgFormat(
```

Comment:
Remove

Reply:
Nvm - spoke with Arjun, it is okay to use resource group region if not provided by user. As long as it tells user this behavior using --help command, which it does:

![image](https://github.com/user-attachments/assets/dabf3d6c-af22-442b-bfa9-ef0e879d3118)